docs/contributing.rst

```diff
@@ -8,46 +8,114 @@ For a more practical guide to the tedana development, please see our
 in-person workshops or development sprints, and when giving talks about the software.
 
 As stated in the code, severe or repeated violations by community members may result in exclusion
 from collective decision-making and rejection of future contributions to the ``tedana`` project.
 
-.. _code of conduct: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
+.. _The full code of conduct is here: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
+
+Scope of tedana
+```````````````
+tedana is a collection of tools, software and a community related to echo time
+(TE) dependent analyses. The umbrella of tedana covers a number of overlapping,
+but somewhat distinct ideas related to multi-echo analysis. This scope covers
+collecting multi-echo data (Acquisition), combining those echoes together
+(Combining), with optional noise removal (Denoising), inspecting the outputs
+(Visualization) and answering multi-echo related questions (Community). In
+general, preprocessed data is pushed through tedana, producing outputs that
+are ready for further analyses. 
+
+Acquisition
+-----------
+
+While the development of multi-echo sequences is far beyond the current scope
+of tedana, the tedana community is committed to providing guidelines on current
+multi-echo implementations. This will include both specific instructions for
+how to collect multi-echo data for multiple vendors as well as details about
+what types of data have been collected thus far. These details are subject to
+change, and are intended to provide users with an idea of what is possible,
+rather than definitive recommendations.
+
+Our focus is on functional MRI, including both magnitude and phase data,
+however we understand that quantitative mapping has the potential to aid in
+data processing. Thus, we believe that some details on non-functional MRI
+acquisitions, such as detailed T2* mapping, may fall within the scope of
+tedana. Acquisition related details can be found in the tedana Documentation.
+
+Combining echoes
+----------------
+
+An early step in processing data collected with multiple echoes is the
+combination of the data into a single time series. We currently implement
+multiple options to combine multi-echo data and will add more as they continue
+to be developed. This is an area of active development and interest.
+
+Denoising
+---------
+
+tedana was developed out of a package known as multi-echo ICA (ME-ICA or meica)
+developed by Dr. Prantik Kundu. Though the usage of ICA for classification of
+signal vs noise components has continued in tedana, this is not a rule. The
+tedana community is open and encouraging of new methods, whether or not they
+have a basis in ICA. 
+
+We are interested in any method that seeks to use the information from multiple
+echoes to identify signal (defined here as BOLD signals arising from neural
+processing) and noise (defined here as spurious changes unrelated to neural
```

Comment:
Just since this signal can actually be quite structured :point_down: 

```suggestion
processing) and noise (defined here as changes unrelated to neural
```